### PR TITLE
fix: add freeform post type

### PR DESCRIPTION
**Pull request by rebelchris**
Added support for freeform post type.

- Re-uses image field
- New fields content and contentHtml as described in PR

Mutation is a separate ticket.

WT-1285 #done

### CODE REVIEW

**Review by capJavert — APPROVED** [review]

File: src/entity/posts/FreeformPost.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { ChildEntity, Column } from 'typeorm';
+import { Post, PostType } from './Post';
+
+@ChildEntity(PostType.Freeform)
+export class FreeformPost extends Post {
+  @Column({ type: 'text', nullable: true })
+  image?: string;
+
+  @Column({ type: 'text' })
+  content: string;
```
Comment (idoshamun) [inline src/entity/posts/FreeformPost.ts:10]: From what I understand from product, this is optional which makes `contentHtml` optional as well

File: src/migration/1682497694917-FreeformPost.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class FreeformPost1682497694917 implements MigrationInterface {
+  name = 'FreeformPost1682497694917';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "post" ADD "content" text`);
+    await queryRunner.query(`ALTER TABLE "post" ADD "contentHtml" text`);
+    await queryRunner.query(
```
Comment (idoshamun) [inline src/migration/1682497694917-FreeformPost.ts:9]: This is not needed I guess

**Review by idoshamun — COMMENTED** [review]
Please check my one comment

File: src/entity/posts/FreeformPost.ts (hunk 1 above)
Comment (rebelchris) [reply src/entity/posts/FreeformPost.ts:10]: But that is the whole type right?
What else is there just a title alone?

File: src/migration/1682497694917-FreeformPost.ts (hunk 2 above)
Comment (rebelchris) [reply src/migration/1682497694917-FreeformPost.ts:9]: I needed it locally the view needs a update if it exists.
Unless there's an alternative way of refetching it?

**Review by idoshamun — APPROVED** [review]

**Pull request merged**